test/models/subscription_status_test.rb

```diff
@@ -37,10 +41,15 @@ def test_to_status_invalid
     end
 
     def test_to_status_unsubscribed_hypervisor
-      stub_status('unsubscribed_hypervisor')
+      host.subscription_facet.stubs(:unsubscribed_hypervisor?).returns(true)
       assert_equal Katello::SubscriptionStatus::UNSUBSCRIBED_HYPERVISOR, status.to_status
     end
 
+    def test_sca_enabled_hypervisor
```

Comment:
nitpick: since we now check for SCA at the top of the status method, the host type (hypervisor or not) is irrelevant. so, would prefer test_sca_enabled here

Reply:
> looks good! i know the testing was a bit wild with this one, so you might wanna check that all the stubs added now are actually necessary if you didn't already, though I suspect they're valid even after all the changes

Sounds good, will address the nit and test commenting out the stubs and see what fails.